src/content/index.ts

```diff
@@ -0,0 +1,47 @@
+import browser from 'webextension-polyfill'
+import type { MessageType } from '../types'
+
+const KEY = 'LOCALE_SWITCHER_LANGUAGE'
+
+function embedScript() {
+  const locale = window.localStorage.getItem(KEY)
+  if (!locale) return
+
+  const code = `
+    (() => {
+      Object.defineProperties(window.navigator.__proto__, {
```

Comment:
I previously used `Navigator.prototype` here. Is there a difference between that and this?

Reply:
Mhh not really, but since most libs take `window.navigator` as the source I thought it might be a good idea to use that. Both seem to work, however they are not the same object according to `===`. Quite werid, but works